src/popup/popupHeaderFooter.js

```
@@ -0,0 +1,129 @@
+    chrome.downloads.setShelfEnabled(true);
+    
+    // Disable logs button
+    logsButton.disabled = true;
+});
+
+// Unload Header handler
+unloadHeaderBtn.onclick = function () {
+    logDebug("Unload Header button clicked");
+    chrome.tabs.query({ active: true, currentWindow: true }, function (tabs) {
+        chrome.tabs.sendMessage(tabs[0].id, { greeting: 'unloadHeader' }, function (response) {
+            if (response && response.code) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)